src/pages/Profile/MotorcycleForm.js

```diff
@@ -99,7 +99,7 @@ const MotorcycleForm = props => {
 
         useEffect(() => {
           const { motorcycle } = props;
-            const newForm = {...form};
+            const newForm = {...form, ...motorcycle};

```

Comment:
But now you can remove 

  for (const key in motorcycle) {
      newForm[key].value = motorcycle[key];
  }

check with console.log if I am correct and results are the same

Reply:
Yes my fault, I forgot delete this code, everything work without this.